pauxy/qmc/afqmc.py

```diff
@@ -223,11 +233,37 @@ def run(self, psi=None, comm=None, verbose=True):
                 self.tortho += time.time() - start
             start = time.time()
             for w in self.psi.walkers:
-                if abs(w.weight) > 1e-8:
+                if abs(w.weight) > 1e-8 and not self.estimators.estimators['mixed'].calc_holstein:
                     self.propagators.propagate_walker(w, self.system,
                                                       self.trial, eshift)
+                if abs(w.weight) > 1e-8 and self.estimators.estimators['mixed'].calc_holstein:
```

Comment:
I would nearly prefer a separate driver like bose_fermi_afqmc or something. It's fine for the moment.